lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -362,9 +389,7 @@ putSignedTx inputs outputs witnesses = do
   where
     -- Assumes the `TxWitness` has been faithfully constructed
```

Comment:
If we defined `TxWitness` here (i.e target-specific) we could hide the constructor… But 🤷‍♂